README.md

```diff
@@ -1 +1,42 @@
-# angular-restricting-directive-usage-readme
\ No newline at end of file
+# Restricting Directive usage
+
+## Overview
+
+We've learned that directives can be used as HTML attributes and HTML elements, but did you know you can tell Angular to only allow your directive to be used in a certain way?
+
+## Objectives
+
+- Describe the syntaxes for declaring Directives
+- Restrict our Directive
+
```

Comment:
Add a section here that demonstrates the need to restricting usage. This section should go over

1. The four different ways we can use directives, with code examples. 
2.  The problems with not restricting our directive usage. This would be a good place to talk about why it's not recommended to use comments and classes.